builder_test.go

```diff
@@ -1003,3 +1003,66 @@ func TestSendWithReversePath(t *testing.T) {
 		t.Errorf("msg bytes did not contain text body %q", text)
 	}
 }
+
+func TestEmptyTo(t *testing.T) {
+	sender := &mockSender{}
+	from := "from@example.com"
+	text := []byte("test text body")
+	ret := "return@example.com"
+	a := enmime.Builder().
+		Text(text).
+		From("name", from).
+		Subject("foo").
+		To("", "")
```

Comment:
Let's put a value in the name section to prove we are testing for emtpy addr only.